Maquetado/index.html

```diff
@@ -11,5 +11,13 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1> 
+  <div class="tech-border">
```

Comment:
this can be achieved by adding a `::before` pseudoelement to the tech without adding extra elements in the DOM